public/utils/helpers.js

```diff
@@ -47,6 +47,22 @@ export const inputLimitText = (
   );
 };
 
+export async function deleteMonitor(monitor, httpClient, notifications) {
+  const { id, version } = monitor;
+  const poolType = monitor.item_type === 'composite' ? 'workflows' : 'monitors';
```

Comment:
Non-blocking nitpick: It may be worthwhile to compile an exhaustive list of `item_type`s in a constant somewhere.